graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/user/TopSecondDegreeByCountForUser.java

```diff
@@ -45,6 +46,13 @@ public TopSecondDegreeByCountForUser(
     StatsReceiver statsReceiver) {
     super(leftIndexedBipartiteGraph, expectedNodesToHit, statsReceiver);
   }
+  protected boolean isEdgeEngagementWithinAgeLimit(
+      TopSecondDegreeByCountRequestForUser request,
```

Comment:
for the first argument, prefer passing in maxEdgeEngagementAgeInMillis instead of request.
for the second argument, prefer passing in currentEdgeEngagementTimeInMillis instead of edgeIterator.